code/modules/mob/living/simple_animal/friendly/koala.dm

```diff
@@ -0,0 +1,21 @@
+//Koala
+/mob/living/simple_animal/koala
```

Comment:
I object to this on the grounds that a realistic koala is a horrible chlamydia-infected screaming monstrosity.